test/ui/ui-main.test.ts

```diff
@@ -131,6 +132,34 @@ describe("UI tests - A user can swap and mint tokens", () => {
     expect(swapped).toBeTruthy();
   });
 
+  it("As a User I can burn all liquidity MGA - mETH", async () => {
+    await testUser1.refreshAmounts(AssetWallet.BEFORE);
+    const amountToMint = new BN(visibleValueNumber).div(new BN(2000));
+    await testUser1.mintLiquidity(ETH_ASSET_ID, MGA_ASSET_ID, amountToMint);
+    const mga = new Mangata(driver);
+    await mga.navigate();
+    const sidebar = new Sidebar(driver);
+    await sidebar.clickOnLiquidityPool(MGA_ASSET_NAME, mETH_ASSET_NAME);
+    await sidebar.clickOnRemoveLiquidity();
+    const modal = new BrunLiquidityModal(driver);
+    await modal.setAmount("100");
```

Comment:
created the test about the 100% asset removal. 
IS now passing, so I think that we need to change the minting values to have some small decimals, right @Andkoo ?

Reply:
Can you please elaborate on why is that so? The test should be passing since removing 100% liq has been already fixed in the FE.

Reply:
ok! gogogogogo

Reply:
Added some decimals,
before:
5 kunits 
now
5.123456789123456